docs/nodes/light-node.mdx

```diff
@@ -10,10 +10,10 @@ import blockspaceraceVersions from "../../versions/blockspacerace_versions.js";
 
 # Setting up a Celestia light node
 
-````mdx-code-block
+```mdx-code-block
```

Comment:
my editor auto fixed these

Reply:
just will make sure it builds fine, should be good. thank you!